manifests/profile/www_lib/vhosts/fulcrum.pp

```diff
@@ -9,10 +9,22 @@
 # @example
 #   include nebula::profile::www_lib::vhosts::fulcrum
 class nebula::profile::www_lib::vhosts::fulcrum (
-  String $docroot = '/hydra/heliotrope-production/current/public'
+  String $docroot = '/fulcrum/app/current/public',
```

Comment:
Exactly. It allows us to change the "default", in the literal sense that the argument has a new default value, but also in the broader sense of the app-oriented paths being what we would expect to see, rather than infrastructure-oriented paths. That change requires any "legacy" servers to get hieradata to preserve the use of their now-deprecated paths (which has been completed separately).